LibreNMS/Alert/Transport/Twilio.php

```diff
@@ -0,0 +1,113 @@
+
+use LibreNMS\Alert\Transport;
+
+class Twilio extends Transport
+{
+    public function deliverAlert($obj, $opts)
+    {
+        if (empty($this->config)) {
+            return $this->deliverAlertOld($obj, $opts);
+        }
+        $twilio_opts['sid']  = $this->config['twilio-sid'];
+        $twilio_opts['token'] = $this->config['twilio-token'];
+        $twilio_opts['sender'] = $this->config['twilio-sender'];
+        $twilio_opts['to']    = $this->config['twilio-to'];
+        return $this->contacttwilio($obj, $twilio_opts);
+    }
+
+    public function deliverAlertOld($obj, $opts)
+    {
+        return $this->contactsmsfeedback($obj, $opts);
+    }
+
+    public static function contacttwilio($obj, $opts)
+    {
+        $params = array(
+            'sid' => $opts['sid'],
+            'token' => $opts['token'],
+            'phone' => $opts['to'],
+            'text' => $obj['title'],
+            'sender' => $opts['sender'],
+        );
+
+	$url    = 'https://api.twilio.com/2010-04-01/Accounts/' . $params['sid'] . '/Messages.json';
+
+	$data = array(
+		'From' => $params['sender'],
+		'Body' => $params['text'],
+		'To' => $params['phone'],
+	);
+	$post = http_build_query($data);
+
+        $curl   = curl_init($url);
+
+
+        // set_curl_proxy($curl);
+
+	curl_setopt($curl, CURLOPT_POST, true);
+	curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+	curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+	curl_setopt($curl, CURLOPT_HTTPAUTH, CURLAUTH_BASIC);
+	curl_setopt($curl, CURLOPT_USERPWD, $params["sid"]. ":" . $params["token"]);
+	curl_setopt($curl, CURLOPT_POSTFIELDS, $post);
+
+	$ret = curl_exec($curl);
+
+	if (curl_getinfo($curl,CURLINFO_RESPONSE_CODE))
+		return true;
+    }
+
+    public static function configTemplate()
+    {
+        return [
+            'config' => [
+                [
+                    'title' => 'SID',
+                    'name' => 'twilio-sid',
+                    'descr' => 'Twilio SID',
+                    'type' => 'text',
+                ],
+                [
+                    'title' => 'Token',
+                    'name' => 'twilio-token',
+                    'descr' => 'Twilio Account Token',
+                    'type' => 'text',
+                ],
+                [
+                    'title' => 'SMS To Number',
+                    'name' => 'twilio-to',
+                    'descr' => 'Mobile number to SMS',
+                    'type' => 'text',
+                ],
+                [
+                    'title' => 'Twilio SMS Number',
+                    'name' => 'twilio-sender',
+                    'descr' => 'Twilio sending number',
+                    'type' => 'text',
+                ],
+            ],
+            'validation' => [
+                'twilio-sid'    => 'required|string',
+                'twilio-token'    => 'required|string',
+                'twilio-to' => 'required',
+                'twilio-sender' => 'required|string',
```

Comment:
Should this be string if it's a mobile number or can you set it to anything you want?